src/System.Net.Http/src/uap/System/Net/HttpClientHandler.cs

```diff
@@ -645,10 +650,35 @@ private void SetOperationStarted()
         {
             if (!_operationStarted)
             {
-                // Since this is the first operation, we set the proxy and server credentials on the
-                // WinRT filter based on the .NET handler's property settings.
+                // Since this is the first operation, we set all the necessary WinRT filter properties.
                 SetFilterProxyCredential();
                 SetFilterServerCredential();
+                if (_serverCertificateCustomValidationCallback != null)
+                {
+                    // The WinRT layer uses a different model for the certificate callback. The callback is
+                    // considered "extra" validation. We need to explicitly ignore errors so that the callback
+                    // will get called.
+                    //
+                    // In addition, the WinRT layer restricts some errors so that they cannot be ignored, such 
```

Comment:
I agree the documentation is not complete in all WinRT areas.

For example, trying to do this:

```c#
var filter = new Windows.Web.Http.Filters.HttpBaseProtocolFilter();
filter.IgnorableServerCertificateErrors.Add(ChainValidationResult.RevocationInformationMissing);
                filter.IgnorableServerCertificateErrors.Add(ChainValidationResult.Revoked);
```
results in this exception:

>System.ArgumentException: The parameter is incorrect.
'item': Provided value is not an ignorable ChainValidationResult value.